jest.config.js

```diff
@@ -20,12 +20,12 @@ module.exports = {
       // https://github.com/carbon-design-system/carbon-charts/issues/594
       functions: 60,
     },
-    './src/components/Table/TableHead/TableHead.jsx': {
-      statements: 73,
-      branches: 77,
-      lines: 73,
-      functions: 57,
-    },
+    // './src/components/Table/TableHead/TableHead.jsx': {
+    //   statements: 73,
+    //   branches: 77,
+    //   lines: 73,
+    //   functions: 57,
+    // },
```

Comment:
This should either be removed entirely or reinstated with proper threshold exception numbers.